tests/cases/conformance/types/contextualTypes/jsxAttributes/contextuallyTypedStringLiteralsInJsxAttributes01.tsx

```diff
@@ -2,9 +2,12 @@
 // @declaration: true
 
 namespace JSX {
-    interface IntrinsicElements {
+    export interface IntrinsicElements {

```

Comment:
The test was invalid because `IntrinsicElements` wasn't actually exposed to the code below